lua/virt-column/init.lua

```diff
@@ -74,10 +73,10 @@ M.refresh = function()
 
     M.clear_buf(bufnr)
 
-    for i = 1, #lines, 1 do
+    for i = 1, vim.fn.line "$", 1 do
```

Comment:
```suggestion
    for i = 1, vim.api.nvim_buf_line_count(bufnr), 1 do
```

This can be a lua function

Reply:
Good catch 👍🏼 I fixed this.